### PR TITLE
Correct links to packages info at v4 website

### LINKED ISSUE #5405: 4.x Doc links update
Update doc links in README for all 4.x packages: 
as current link used is invalid  http://web3js.readthedocs.io/en/4.0/

### PR DESCRIPTION
**Pull request by ai-slave**
## Description

Replaced wrong links to correct ones pointing to v4 website.

Fixes #5405

## Type of change

- [+] Bug fix (non-breaking change which fixes an issue)

## Checklist for 1.x:

- [+] I have selected the correct base branch.
- [+] I have performed a self-review of my own code.
- [+] I have commented my code, particularly in hard-to-understand areas.
- [+] I have made corresponding changes to the documentation.
- [+] My changes generate no new warnings.
- [+] Any dependent changes have been merged and published in downstream modules.
- [+] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [+] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [+] I ran `npm run build` with success.
- [+] I have tested the built `dist/web3.min.js` in a browser.
- [+] I have tested my code on the live network.
- [+] I have checked the Deploy Preview and it looks correct.
- [+] I have updated the `CHANGELOG.md` file in the root folder.

## Checklist for 4.x:

- [+] I have selected the correct 4.x base branch.
- [+] Within the description, the feature or issue is discussed in detail or an issue is linked.   
- [+] I have performed a self-review of my own code
- [+] I have commented my code, particularly in hard-to-understand areas.
- [+] I have added any required tests for the changes I made 
- [+] I have made corresponding changes to the documentation
- [+] Any dependent changes have been merged and published in downstream modules.
- [+] I ran `yarn` successfully
- [+] I ran `yarn lint` successfully
- [+] I ran `yarn build:web` successfully
- [+] I ran `yarn test:unit` successfully
- [+] I ran `yarn test:integration` successfully
- [+] I ran `compile:contracts` successfully
- [+] I have tested my code.
- [+] I have updated the corresponding `CHANGELOG.md` file in the packages I have edited.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,23 +34,23 @@ yarn add web3
...
+| [web3-eth-personal](https://github.com/ChainSafe/web3.js/tree/4.x/packages/web3-eth-personal)     | [![npm](https://img.shields.io/npm/v/web3)](https://www.npmjs.com/package/web3-eth-personal)   | [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) | [![documentation](https://img.shields.io/badge/typedoc-blue)](https://docs.web3js.org/api/web3-eth-personal) | Module to interact with the Ethereum blockchain accounts stored in the node                                   |
+| [web3-eth-tx](https://github.com/ChainSafe/web3.js/tree/4.x/packages/web3-eth-tx)                 | [![npm](https://img.shields.io/npm/v/web3)](https://www.npmjs.com/package/web3-eth-tx)         | [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) | [![documentation](https://img.shields.io/badge/typedoc-blue)](https://docs.web3js.org/api/web3-eth-tx) | [@ethereumjs/tx](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/tx) wrapper class     |
```
Comment (jdevcs) [inline README.md]: we dnt have eth-tx package so this line should be removed: 
image

**Review by jdevcs — CHANGES_REQUESTED** [review]
@ndrozd Thanks for your contribution, need a change in your PR.

**Comment by ai-slave** [comment]
@jdevcs done

**Review by jdevcs — APPROVED** [review]

**Review by Muhammad-Altabba — APPROVED** [review]
Thanks for your contribution.

**Review by luu-alex — APPROVED** [review]

**Comment by jdevcs** [comment]
@ndrozd The base branch requires all commits to be signed.

**Comment by ai-slave** [comment]
Sorry I don't get what's my action item here

**Comment by luu-alex** [comment]
@ndrozd to contribute, your commits need to be verified. https://github.blog/changelog/2022-08-23-ssh-commit-verification-now-supported/

**Review by avkos — APPROVED** [review]

**Comment by ai-slave** [comment]
Well it seems like I've messed something up while signing my prev commits....

**Comment by luu-alex** [comment]
If it makes it easier, first verify your commits and then you can create a new branch and do the exact same commits

**Comment by ai-slave** [comment]
It seems like readme file is totally different now anyway

**Pull request closed without merging**